test/invoker_test.rb

```diff
@@ -167,7 +167,10 @@
     begin
       warnings = nil
       redirect_streams do |out, err|
-        invoke_cli_to_buffer(%w(-w -o /dev/null), '-') { $NO_SUCH_VARIABLE || 'text' }
+        invoke_cli_to_buffer(%w(-w -o /dev/null), '-') {
+          A_CONST = 10  
+          A_CONST = 20
```

Comment:
I like this way of triggering a warning. I think this should be moved to master now.

Reply:
I'd support merging this PR to master rather :)
It's been a long time and lots of efforts, thank you @Mogztter and @mojavelinux!

Reply:
I certainly support merging this PR to master once we can verify all the tests are passing. What I'm talking about here is that I like this technique for triggering a warning because we know it is very portable.